src/ui/public/vislib/isPercentage.js

```diff
@@ -0,0 +1,10 @@
+import { SCALE_MODES } from '../vis/scale_modes';
+
+export function isPercentage(aggConfigId, seriesParams, valueAxes) {
```

Comment:
this is already implemented on the value axis. 
`handler.charts[0].series[0].getValueAxis().axisConfig.isPercentage()`

what about adding a getSeries method to handler, which will return the right series based on id ?
`handler.getSeries(serieId).getValueAxis().axisConfig.isPercentage()`